tools/wptserve/wptserve/request.py

```diff
@@ -541,7 +546,7 @@ def add(self, name, value):
 
     def __getitem__(self, key):
         """Get the first value with a given key"""
-        #TODO: should this instead be the last value?
+        # TODO: should this instead be the last value?
```

Comment:
Just remove the comment since we didn't change this in N years.

Reply:
Can we add an assert here that the passed in value is binary? And maybe in the other accessor methods too.

Reply:
Note that the accessors in `RequestHeaders` call `_maybe_encode(key)` instead. Shall we make them consistent?